cli/cmd/env/env.go

```diff
@@ -0,0 +1,324 @@
+func SetCmd(cfg *config.Config) *cobra.Command {
+	var projectName string
+	setCmd := &cobra.Command{
+		Use:   "set <key> <value> --project <project name>",
+		Args:  cobra.ExactArgs(2),
+		Short: "set variable",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			key := args[0]
+			value := args[1]
+			client, err := cmdutil.Client(cfg)
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+
+			ctx := cmd.Context()
+			resp, err := client.GetProjectVariables(ctx, &adminv1.GetProjectVariablesRequest{
+				OrganizationName: cfg.Org,
+				Name:             projectName,
+			})
+			if err != nil {
+				return err
+			}
+
+			if val, ok := resp.Variables[key]; ok && val == value {
+				return nil
+			}
+
+			if resp.Variables == nil {
+				resp.Variables = make(map[string]string)
+			}
+			resp.Variables[key] = value
+			updateResp, err := client.UpdateProjectVariables(ctx, &adminv1.UpdateProjectVariablesRequest{
+				OrganizationName: cfg.Org,
+				Name:             projectName,
+				Variables:        resp.Variables,
+			})
+			if err != nil {
+				return err
+			}
+
+			cmdutil.SuccessPrinter("Updated project variables\n")
+			tableprinter.PrintHeadList(os.Stdout, variable.Serialize(updateResp.Variables), "Project Variables")
+			return nil
+		},
+	}
+
+	setCmd.Flags().StringVar(&projectName, "project", "", "")
+	_ = setCmd.MarkFlagRequired("project")
+	return setCmd
+}
+
+// RmCmd is sub command for env. Removes the variable for a project
+func RmCmd(cfg *config.Config) *cobra.Command {
+	var projectName string
+	rmCmd := &cobra.Command{
+		Use:   "rm <key> --project <project name>",
+		Args:  cobra.ExactArgs(1),
+		Short: "remove variable",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			key := args[0]
+			client, err := cmdutil.Client(cfg)
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+
+			ctx := cmd.Context()
+			resp, err := client.GetProjectVariables(ctx, &adminv1.GetProjectVariablesRequest{
+				OrganizationName: cfg.Org,
+				Name:             projectName,
+			})
+			if err != nil {
+				return err
+			}
+
+			if _, ok := resp.Variables[key]; !ok {
+				return nil
+			}
+
+			delete(resp.Variables, key)
+			update, err := client.UpdateProjectVariables(ctx, &adminv1.UpdateProjectVariablesRequest{
+				OrganizationName: cfg.Org,
+				Name:             projectName,
+				Variables:        resp.Variables,
+			})
+			if err != nil {
+				return err
+			}
+
+			cmdutil.SuccessPrinter("Updated project \n")
+			tableprinter.PrintHeadList(os.Stdout, variable.Serialize(update.Variables), "Project Variables")
+			return nil
+		},
+	}
+	rmCmd.Flags().StringVar(&projectName, "project", "", "")
+	_ = rmCmd.MarkFlagRequired("project")
+	return rmCmd
+}
+
+func ShowEnvCmd(cfg *config.Config) *cobra.Command {
+	var projectName string
+	showCmd := &cobra.Command{
+		Use:   "show --project <project name>",
+		Short: "show variable for project",
```

Comment:
1. Capitalize and remove flags
2. Make text "Show credentials and other variables"